tools/mo/unit_tests/moc_tf_fe/conversion_basic_models_test.py

```diff
@@ -241,11 +241,8 @@ def test_conversion_failure_fallback_default(self):
 
     @unittest.skipIf(platform == 'darwin', reason="Ticket - 122182")
     def test_conversion_failure_fallback_use_new_frontend(self):
```

Comment:
Can we remove unittest.skipIf here? 
And may be rename the test, as it does not fail anymore.

Reply:
Fixed.